sdk/core/AzureCore/Source/DataStructures/CancellationToken.swift

```diff
@@ -29,7 +29,7 @@ import Foundation
 public final class CancellationToken: Codable, Equatable {
     public private(set) var isCanceled: Bool
     public private(set) var isStarted: Bool
-    public private(set) var timeoutInSeconds: Double?
+    public var timeoutInSeconds: Double?
```

Comment:
We should revert this and add an internal method to set the timeout. We don't want a customer to be able to change the timeout value, because changing it while the timer is running could lead to confusion.

Reply:
Fixed.